docs/example-scenario/data/realtime-analytics-vehicle-iot.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Real time IoT data processing on Azure
```

Comment:
How about something like "Ingestion and Processing of real-time automotive IOT data"?

Reply:
Sounds good to me, suits better here, thanks for the recommendation. I will change it and commit back. Please let me know if you have any other thoughts.